exercises/practice/word-search/lib/word_search.ex

```diff
@@ -0,0 +1,14 @@
+defmodule WordSearch do
+  defmodule Location do
+    defstruct [:from, :to]
+    @type t :: %Location{from: {integer, integer}, to: {integer, integer}}
```

Comment:
Secondary to that comment in the test file, we should either standardize this to `{x, y}` or find a way to communicate an alternative meaning to the tuple pair